### PR TITLE
Chart adjustments (#53).

### LINKED ISSUE #53: [Corrections] Minor Chart Adjustments
- When only a single data point is available, show a single dot (currently invisible without hovering). 
- Numbers in the tooltips should have commas to separate the thousands (i.e., “1,000,000” instead of “1,000”).
- Tooltip should include the date as "[Month] [Year]" (e.g., February 2020). See image below. 
Group 61
- Numbers on the y-axis should have commas to separate the thousands (i.e., “1,000,000” instead of “1,000”).
- When the net change is 0, the resulting % in the legend should be “0%” not “+0%”. 
- Upper padding of the chart on mobile is too tight 
- Lines and points should still be drawn for zero values (see below example, Wisconsin:Probation Revocations:New Crime). 
image
- "Not Available" label should have a white background and be centered vertically and horizontally in the chart canvas.

### PR DESCRIPTION
**Pull request by Wmaileq**
## Description of the change

Adjustments on:
- chart tooltips
- chart `not available` state
- case when metric has only one data point 
- "0%" and "+0%" issue
- thousand separators

## Type of change

- [X] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #53.

### Development

These boxes should be checked by the submitter prior to merging:

- [X] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by jessex** [comment]
@hobuobi could you also do a quick test on this one, and respond to the comment from Vlad above? He needs fixture data to reproduce one of the issues you called out. I've done a quick review and I think he's solved all of the issues that are testable from my end but I want you to sign off.

**Review by jessex — CHANGES_REQUESTED** [review]
@Wmaileq I notice one issue from my testing: the years in the chart tooltips are only 2 digits instead of 4. We want 2019, 2020, etc., instead of 19, 20, etc.

Screen Shot 2021-02-16 at 09 46 42


Screen Shot 2021-02-16 at 09 46 49

File: src/components/Chart/Chart.js [hunk 1]
```diff
@@ -148,7 +152,14 @@ const Chart = ({ title, hint, chartData }) => {
               },
               tooltips: {
                 callbacks: {
-                  title: () => null,
+                  title: (item) => {
+                    const [month, year] = item[0].label.split("/");
+                    return `${months[month - 1]} ${2000 + toInt(year)}`;
```
Comment (jessex) [inline src/components/Chart/Chart.js:157]: Isn't the year that's coming in via the data.json file already 4 digits? Why would we need to do this? If it's stripped of the left-most 2 digits at some point for some reason, we should consider changing that instead. It's conceivable that we'll have data that dates back to before 2000 and this logic would generate years like 2096 and 2099.

**Review by jessex — CHANGES_REQUESTED** [review]

**Comment by hobuobi** [comment]
Almost everything looks great! The only thing is that I still see the +0% issue (for example, the Probation Population on US_ND).

File: src/components/Chart/Chart.js (hunk 1 above)
Comment (Wmaileq) [reply src/components/Chart/Chart.js]: @jessex Good point, fixed

**Comment by Wmaileq** [comment]
Finally i was able to reproduce Collin's issue. I think now it is ready to be shipped.

**Review by jessex — APPROVED** [review]
Confirmed with the latest real data file locally that the 0% legend issue is fixed (confirmed via US_ND) and that the missing tooltip entries for 0 values is also fixed (confirmed via US_WV). We're all set here! Thanks, @Wmaileq

**Pull request merged**